src/material/radio/radio.scss

```diff
@@ -66,14 +66,20 @@ $ripple-radius: 20px;
 
 // The inner circle for the radio, shown when checked.
 .mat-radio-inner-circle {
+  $transition-duration: 280ms;
+  $base-transition: transform ease $transition-duration, background-color ease $transition-duration;
   border-radius: 50%;
   box-sizing: border-box;
   display: block;
   height: $size;
   left: 0;
   position: absolute;
   top: 0;
-  transition: transform ease 280ms, background-color ease 280ms;
+  // On some zoom levels the `scale(0.001)` from below can cause the circle to be shown as a 1x1
+  // dot (see #22036). Ensure that it's hidden using `opacity`. There's a slight transition with
+  // a long delay so that switching the opacity only applies after the `transform` is done.
+  opacity: 0;
```

Comment:
I believe JAWS + IE11 treats `opacity: 0` as hidden, unfortunately.

Reply:
This is just a blank `span`, the `input` is a sibling element.